app/assets/v2/js/grants/index.js

```diff
@@ -162,6 +162,7 @@ if (document.getElementById('grants-showcase')) {
       page: 1,
       collectionsPage: 1,
       limit: 6,
+      show_active_clrs: window.localStorage.getItem('show_active_clrs') != 'false',
```

Comment:
false string? seems to me this will be === true

Reply:
maybe you should use JSON.parse(window.localStorage.idx_status = 'false')
 and also handle when there is not localstorage show_active_clrs to avoid errors

Reply:
Anyway works 
default null  >  true
'false' > false
anything > true 
but in general I feel is better when we deal with booleans to keep it booleans